src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php

```diff
@@ -11,11 +11,18 @@
 
 abstract class AbstractAsciiTestCase extends TestCase
 {
-    protected static function createFromString(string $string): AbstractString
+    protected static function stringClass(): string
```

Comment:
Is this method needed? If not, I'm not sure it's better as it creates more indirections.

Reply:
It is used in `testWrap` for the static call.

Reply:
I could use `$class = get_class(static::createFromString(''))` instead.

Reply:
Or directly: `$s = static::createFromString(''); $s::wrap(...);`